hdr_html_canvas_element.md

```diff
@@ -411,3 +462,63 @@ Arguably, the HDR configuration data could be attached to the `CanvasRenderingCo
 
 The HDR configuration data should travel with an `ImageBitmap` when displayed in an `ImageBitmapRenderingContext`.
 That may inform where we should put this.
+
+### Are the new `ScreenAdvanced` attributes a fingerprinting vector?
```

Comment:
Move to GitHub issue

Reply:
If this is a W3C explainer (which it kind of is), then it needs to address privacy concerns within the document. So the question should be on GitHub for discussion, but it also should be in the document so that the answer gets filled out eventually.

Reply:
Added a section for this